systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java

```diff
@@ -278,14 +273,14 @@ public SetupClusterOperator runManualOlmInstallation(final String fromOlmChannel
     private void helmInstallation() {
         LOGGER.info("Install ClusterOperator via Helm");
         helmResource = new HelmResource(namespaceInstallTo, namespaceToWatch);
-        createCONamespaceIfNeeded();
+        createClusterOperatorNamespaceIfPossible();
```

Comment:
WHat will happen when the creation won't be possible? 😃

Reply:
It will be impossible only when we use two CO's in one namespace (so the second CO will not re-crease the already deployed namespace and remove the first one). That's currently one use case for this logic AFAIK.

Reply:
so do we need `IfPossible` in the method name?